source/tests/mock-kraken/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+include_directories("..")
+
+set(ALL_LIBS workers ed disruption_api calendar_api time_tables types autocomplete proximitylist
+    ptreferential time_tables data routing fare types georef utils SimpleAmqpClient
+    rabbitmq-static log4cplus ${Boost_LIBRARIES} protobuf ${NAVITIA_ALLOCATOR})
+
+# every tests will produce an executable used in jormungandr integration tests
+
+add_executable(main_routing_test main_routing_test.cpp)
```

Comment:
You will probably have to update this https://github.com/CanalTP/navitia-docker-compose/blob/master/builder/build.sh#L80 and this https://github.com/CanalTP/navitia-docker-compose/blob/master/builder/Dockerfile-mock-kraken#L26

Reply:
https://github.com/CanalTP/navitia-docker-compose/pull/54